guides/common/modules/proc_configuring-satellite-for-outgoing-emails.adoc

```diff
@@ -5,6 +5,7 @@ To send email messages from {ProjectServer}, you can use either an SMTP server,
 
 .Prerequisites
 
+* Postfix from {RHEL} is the recommended email server to use. Some SMTP servers with anti-spam protection or grey-listing features are known to cause problems, to setup outgoing email with such service either install and configure postfix on {ProjectServer} for relay or use `sendmail` command instead.
 * If you have upgraded from a previous release, rename or remove the configuration file `/usr/share/foreman/config/email.yaml` and restart the `httpd` service. For example:
```

Comment:
Unrelated, this "previous release" was Foreman 1.15 and 1.16 removed it. Since https://github.com/theforeman/puppet-foreman/commit/e6ead4595d3008040e56a55cc40a424ee5cb87d7 the installer removes it. No user will ever see this. I opened https://github.com/theforeman/foreman-documentation/pull/334 to get rid of it.